src/vcf2json.nim

```diff
@@ -20,10 +21,21 @@ proc `%`(s: int): JsonNode =
   # Overload JsonNode from int
   # Important for converting
   # '.' to null values
-  if s == int.low:
+  ## When a format field has multiple values per sample, missing values are represeted by 
+  ## int32.low OR (int32.low +1). I can not understand this at all? 
```

Comment:
I'm not sure about the int32.low + 1 but I am guessing this code works?

Reply:
It works... but it is not exactly pretty. After implementing this I opened an hts-nim issue to ask what the expected behaviour was (https://github.com/brentp/hts-nim/issues/57). Thanks to Brent, I now understand the problem and can see a more general way of addressing it.

I will give it a go at using something along the linens of what Brent suggested to clean this up and also handle missing values for types that are not explicitly handled at the moment.

Reply:
Will use dev in the future (sorry).  
Cheers.